### PR TITLE
Adding a CONTRIBUTING.md file 

### PR DESCRIPTION
**Pull request by mynkdhmn**
I have added a simple CONTRIBUTING.md file to this repository. Feel free to suggest any changes.

### CODE REVIEW

**Comment by arshadkazmi42** [comment]
@mynkdhmn This looks great, loved it. Just few changes.
Thank you for working on this 👍 

I was thinking to keep CONTRIBUTING.md and Contributors.md seperate.
As we might by improving CONTRIBUTING.md some time later, keeping Contributors.md serpately will be good idea.
So for this, you need to create new file `Contributors.md` which will have contributors list
and keep the other CONTRIBUTING related changes in the CONTRIBUTING.md

Also, do tag the issue you are working on in your PR description and try to create feature branches when you are working on something, its good practice to have different branches.
For feature branches, you have do it from next PR onwards, this will be fine. Just make the above changes and it will be ready to land.

**Comment by mynkdhmn** [comment]
@arshadkazmi42 
Solved issue regarding adding CONTRIBUTING.md and Contributors.md file.
I have made certain changes. Do suggest any improvements.

**Comment by arshadkazmi42** [comment]
@mynkdhmn This looks good. 
Merging it now. 
Congrats on landing your first PR

File: Contributors.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#### We sincerely thank people who have been activlely contributing to this repository:
+- Arshad Kazmi (https://github.com/arshadkazmi42)
```
Comment (arshadkazmi42) [inline Contributors.md]: It should be exactly in this format

`- [Arshad Kazmi](https://github.com/arshadkazmi42)`

You missed sqaure brackets in the name. I have already merged this, Can you create another PR with this fix?

**Comment by mynkdhmn** [comment]
Sure

**Pull request merged**